tests/tests_pytorch/deprecated_api/test_remove_1-7.py

```diff
@@ -99,15 +99,6 @@ def on_post_move_to_device(self):
         trainer.fit(model)
 
 
-def test_v1_7_0_deprecate_parameter_validation():
-
-    _soft_unimport_module("pytorch_lightning.core.decorators")
```

Comment:
Could you remove the import statement for this function at the top of this file?